### DISCUSSION TITLE
Kubescape scan on k3s cluster

### DISCUSSION THREAD

**Discussion by nickyex2** [kubescape/kubescape#1574] [Q&A]
Would like to ask if kubescape is able to scan k3s clusters. I have no information about this and would like to get a better understanding!

**Comment by matthyx** [comment]
Hi @nickyex2 yes it should work since most of the scanning process happens with the k8s API.
The only component that might have issues is the `hostscanner` if the nodes are very different, you should try it.

**Comment by nickyex2** [comment]
Then are you able to scan a cluster that is on a different IP address or is it just fixed to scanning locally?

**Comment by Oshratn** [comment]
Hey @nickyex2 , indeed you can scan a remote cluster, however, it depends on the default cluster in your kubeconfig file

**Reply by nickyex2** [reply]
oh there is a way to change the default cluster to be scanned?

**Reply by matthyx** [reply]
```
kubescape scan -h
...
  # Scan different clusters from the kubectl context 
  kubescape scan --kube-context <kubernetes context>
```
